diff --git a/javascripts/references.js b/javascripts/references.js
index 0e10c69..7193a6a 100644
--- a/javascripts/references.js
+++ b/javascripts/references.js
@@ -123,6 +123,10 @@ if( typeof jQuery !== 'undefined' ) {
 							$( '#mf-references' ).slideDown();
 						}
 					}
+					// Event callback when reference is shown
+					if(MobileFrontend.references.onReferenceShown) {
+						MobileFrontend.references.onReferenceShown();
+					}
 				} else {
 					close();
 				}
